### PR TITLE
Eth id req update

### LINKED ISSUE #51: Add eth_id to requests
- [ ] Add eth_id to requests table in database
- [ ] Autoincrement integer

### PR DESCRIPTION
**Pull request by perryr16**
## What's this PR do?
* add eth_id (int) to request table
## Where should the reviewer start? 
* `requestView.py` line 17-20
* to migrate:
* `pipenv shell`
* `python3 manage.py db stamp head`
* `python3 manage.py db migrate`
* `python3 manage.py db upgrade`
## How should this be manually tested?
* pytest
## Any background context you want to provide?
* FE would like to keep track of the chronological order of requests associated with a campaign
## What are the relevant tickets?
* close #51

### CODE REVIEW

File: src/views/RequestView.py [hunk 1]
```diff
@@ -10,8 +11,12 @@
 
 @request_api.route('/', methods=['POST'])
 def create():
+  
   try: 
     req_data = request.get_json()
+    # get number of campaign requests
+    campaign = Campaign.get_one_campaign(req_data['campaign_id'])
+    req_data['eth_id'] = len(campaign.requests)
     data = request_schema.load(req_data)
```
Comment (perryr16) [inline src/views/RequestView.py:20]: This is the meat of it. When creating request the FE will send data that includes that campaign_id. get that campaign and determine the number of requests-- that length will be its eth_id. Modify the given data for eth_id and add it to the request data

File: src/tests/test_request.py [hunk 2]
```diff
@@ -181,3 +181,6 @@ def test_one_request(self):
 
     self.assertEqual(self.request2["description"], json_data2["description"])
     self.assertNotEqual(self.request1["description"], json_data2["description"])
+
+    self.assertEqual(0, json_data1["eth_id"])
+    self.assertEqual(1, json_data2["eth_id"])
```
Comment (perryr16) [inline src/tests/test_request.py:186]: tested eth_id

File: src/views/RequestView.py (hunk 1 above)
Comment (takeller) [reply src/views/RequestView.py:20]: We don't need to do 'eth_id' = len(campaign.requests) + 1? Does 'eth_id' start counting at 0 or 1 on the blockchain?

**Review by takeller — APPROVED** [review]

File: src/views/RequestView.py (hunk 1 above)
Comment (AntonioJacksonII) [reply src/views/RequestView.py:20]: I think Jack said it starts at 0

Comment (takeller) [reply src/views/RequestView.py:20]: Cool. That makes sense then. Thanks

**Review by AntonioJacksonII — APPROVED** [review]

**Pull request merged**